man/systemd.network.xml

```diff
@@ -2147,6 +2147,19 @@ Table=1234</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RapidCommit=</varname></term>
+        <listitem>
+          <para>Takes a boolean. The DHCPv6 client can obtain configuration parameters from a DHCPv6 server
+          through a rapid two-message exchange (solicit and reply). When the rapid commit option is set by
+          both the DHCPv6 client and the DHCPv6 server, the two-message exchange is used. Otherwise, the
+          default four-message exchange (solicit, advertise, request, and reply) is used. The two-message
```

Comment:
"default" is a bit misleading, given that our default actually is the two message exchange.

maybe "traditional" or "classic" or so is a better word?

Reply:
Simply dropped "default". Both two- and four-message exchange protocol is defined in RFC3315, which is one of the earliest RFC, if my knowledge is correct. At least, the rapid commit protocol is not new. Hence, "traditional" or "classic" sounds not good for me.